doc/source/tutorial-quickstart-huggingface.rst

```diff
@@ -194,36 +227,104 @@ and then using the :code:`flwr.server.start_server` function:
         evaluate_metrics_aggregation_fn=weighted_average,
     )
 
-    # Start server
-    fl.server.start_server(
-        server_address="0.0.0.0:8080",
-        config=fl.server.ServerConfig(num_rounds=3),
-        strategy=strategy,
-    )
-
-
 The :code:`weighted_average` function is there to provide a way to aggregate the metrics distributed amongst
 the clients (basically this allows us to display a nice average accuracy and loss for every round).
+Next, we set the number of federated learning rounds in `ServerConfig` using the parameter :code:`num_rounds`:
 
-Putting everything together
----------------------------
+.. code-block:: python
+
+    config = ServerConfig(num_rounds=3)
 
-We can now start client instances using:
+Last but not least, we create a :code:`ServerApp` and pass both `strategy` and `config`:
 
 .. code-block:: python
 
-    fl.client.start_client(
-        server_address="127.0.0.1:8080",
-        client=IMDBClient().to_client()
+    app = ServerApp(
+        config=config,
+        strategy=strategy,
     )
 
 
-And they will be able to connect to the server and start the federated training.
+Train the model, federated!
+---------------------------
+
+With both :code:`ClientApps` and :code:`ServerApp` ready, we can now run everything and see federated
+learning in action. First, we run the :code:`flower-superlink` command in one terminal to start the infrastructure. This step only needs to be run once.
+
+.. admonition:: Note
+    :class: note
+
+    In this example, the :code:`--insecure` command line argument starts Flower without HTTPS and is only used for prototyping. To run with HTTPS, we instead use the argument :code:`--certificates` and pass the paths to the certificates. Please refer to `Flower CLI reference <ref-api-cli.html>`_ for implementation details.
```

Comment:
We now use `--ssl-ca-certfile`, `--ssl-certfile`, and `--ssl-keyfile` instead of `--certificate`.

Reply:
Fixed!